app/utils/print/playback/sections/LeadTrusteeBusiness.scala

```diff
@@ -33,13 +32,14 @@ object LeadTrusteeBusiness {
       Some(Seq(AnswerSection(
         headingKey = Some(messages("answerPage.section.leadTrustee.subheading")),
         Seq(
-          stringQuestion(TrusteeOrgNamePage(index), userAnswers, "trusteeBusinessName"),
           yesNoQuestion(TrusteeUtrYesNoPage(index), userAnswers, "leadTrusteeUtrYesNo", name),
+          stringQuestion(TrusteeOrgNamePage(index), userAnswers, "trusteeBusinessName"),
```

Comment:
TrusteeOrgName should be before TrusteeUtrYesNo

Reply:
As per Rachel's feed back:
Lead trustee - The 'Is trustee a UK registered company/' is the first question. 
The TrusteeUtrYesNo renders 'Is trustee a UK registered company?'